app/assets/stylesheets/admin.scss

```diff
@@ -118,3 +118,63 @@
   margin: 0;
   padding: govuk-spacing(2) govuk-spacing(3) govuk-spacing(2) 0;
 }
+
+// X-GOVUK Secondary Navigation
+// Based on https://x-govuk.github.io/govuk-prototype-components/secondary-navigation/
+.x-govuk-secondary-navigation {
```

Comment:
I think we should be using this bit, the other visual difference I mentioned works when we use this directly:

https://github.com/x-govuk/govuk-prototype-components/blob/283481392bd4cd6e39d1a4b10058f25336917e52/dist/govuk-prototype-components.scss#L173C1-L238